types/swagger-jsdoc/index.d.ts

```diff
@@ -44,32 +44,287 @@
 declare function swaggerJSDoc(options?: swaggerJSDoc.Options): object;
 
 declare namespace swaggerJSDoc {
-    interface ApiInformation {
-        description?: string;
+    /**
+     * Open API Specification (OAS) version 3.0 options
+     */
+    interface Oas3Options {
```

Comment:
`OAS3Options` as acronyms should always be capitalized

Reply:
I have made the suggested changes. please review.